weblate/trans/models/component.py

```diff
@@ -1678,7 +1678,7 @@ def do_update(self, request=None, method=None):
         if result:
             # create translation objects for all files
             try:
-                self.create_translations(request=request)
+                self.create_translations(request=request, run_async=True)
```

Comment:
This should no longer raise exception, so try/except block can be removed.

Reply:
I think that `FileParseError` exceptions can still be raised in case `settings.CELERY_TASK_ALWAYS_EAGER` is `True`? Then `_create_translations` is still called directly.

Reply:
That's true, this code-path still exists.